client/src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
i'd prefer not to use bootstrap. a lot of the styling on this page isn't hard to do and you would learn more about css this way.